src/vmm/src/builder.rs

```diff
@@ -162,10 +165,14 @@ fn create_vmm_and_vcpus(
 
         // x86_64 uses the i8042 reset event as the Vmm exit event.
         let reset_evt = vcpus_exit_evt.try_clone().map_err(VmmError::EventFd)?;
+        let i8042 = Arc::new(Mutex::new(I8042Device::new(
+            reset_evt,
+            EventFd::new(libc::EFD_NONBLOCK).map_err(VmmError::EventFd)?,
+        )));
```

Comment:
note: the `I8042Device::new` can be simplified by moving `EventFd` creation inside and asserting on it.

Reply:
done

Reply:
although, I didn't assert. I'm still returning an Error. This code is in the InstanceStart API call path.